### PR TITLE
language java update

### PR DESCRIPTION
**Pull request by thomasr8**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/language-java.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ jobs: # a collection of steps
       _JAVA_OPTIONS: "-Xmx3g"
       GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
     docker: # run the steps with Docker
-      - image: circleci/openjdk:11.0.3-jdk-stretch # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/openjdk:17.0.1 # ...with this image as the primary container; this is where all `steps` will run
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:12-alpine
+      - image: cimg/postgres:14.0.0
```
Comment (felicianotech) [inline jekyll/_cci2/language-java.md:52]: `14.0` instead of `14.0.0`. The latter doesn't exist.

File: jekyll/_cci2/language-java.md [hunk 2]
```diff
@@ -151,11 +151,11 @@ We also use the `environment` key to configure the JVM and Gradle to [avoid OOM
 version: 2
 ...
     docker:
-      - image: circleci/openjdk:11.0.3-jdk-stretch
+      - image: cimg/openjdk:17.0.1
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:12-alpine
+      - image: cimg/postgres:14.0.0
```
Comment (felicianotech) [inline jekyll/_cci2/language-java.md:158]: `14.0` instead of `14.0.0`. The latter doesn't exist.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/language-java.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/language-java.md]: Fixed

File: jekyll/_cci2/language-java.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/language-java.md]: Fixed

**Review by felicianotech — APPROVED** [review]

**Pull request merged**